### DISCUSSION TITLE
When will collections have types?

### DISCUSSION THREAD

**Discussion by amirsaran3** [near/near-sdk-js#268] [Q&A]
I created a project with `create-near-app` command and I am trying to add types to my collections, but it doesn't allow me.

In your current examples you have `LookupMap<CarSpecs>;`.

My version of `near-sdk-js` is `0.5.0`. 

I saw that this is added/fixed in `0.5.0-1` prerelease. 

How can I use typed collections at the moment?

**Comment by volovyks** [comment]
These features will be available on 0.6 (not released yet). You can use `v0.6.0-0` prerelease for now.
